src/veh_interact.cpp

```diff
@@ -2092,6 +2090,15 @@ void veh_interact::display_details( const vpart_info *part )
         }
     }
 
+    if( part->has_flag( VPFLAG_WHEEL ) ) {
+        fold_and_print( w_details, line+3, col_1, column_width, c_white,
```

Comment:
adding local variable like that 
```cata::optional<islot_wheel> whl = item::find_type( part->item )->wheel;```
could be useful for avoiding of double call of ```find_type()``` and could be used as ```whl->diameter``` and ```whl->width```